common/changes/office-ui-fabric-react/tf-masked-eg_2019-05-24-19-08.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "office-ui-fabric-react",
+      "type": "none"
```

Comment:
nit: technically a change to the published package that people consumes since (as opposed to changes to tests which people do not consume). Also helpful to note in release notes in the context of aka.ms/fabricdemo and website that there is now a new example :)

Reply:
Good call out, I made this minor and added some comment here.